app/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "app-path": "^2.2.0",
+    "app-path": "^3.2.0",
```

Comment:
There are no big changes between these two version of the library: it just uses newer JS syntax and some internal changes ([more info](https://github.com/sindresorhus/app-path/compare/v2.2.0...v3.2.0)).